templates/terraform/resource.erb

```diff
@@ -141,7 +141,7 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
     defer mutexKV.Unlock(lockName)
 <%  end -%>
 
-    url, err := replaceVars(d, config, "<%= object.full_create_url -%>")
+    url, err := replaceVars(d, config, "<%= "{{#{object.__product.name}BasePath}}#{object.create_uri}" -%>")
```

Comment:
This is totally an aesthetic nit, but I think the generated code would look nicer if the BasePaths didn't end with a '/', that way it could be`{{ComputeBasePath}}/projects/..`. instead of `{{ComputeBasePath}}projects/...`

Reply:
I agree that it's nicer aesthetically, but in both MM's `base_url` and the client libs the base path contains the trailing `/`, so for consistency I'd like to keep it.

Reply:
Also consistent with `gcloud`: https://github.com/google-cloud-sdk/google-cloud-sdk/blob/257c9c317e477450562b679774d6fb3d3337bded//lib/googlecloudsdk/core/properties.py#L1663-L1666

Reply:
Got it. In that case, can we add a ValidateFunc to ensure they end with a '/'?